test/integration/voldemort/performance/benchmark/CustomizedWorkload.java

```diff
@@ -0,0 +1,243 @@
+package voldemort.performance.benchmark;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Random;
+
+import com.google.common.collect.Lists;
+
+import voldemort.client.StoreClient;
+import voldemort.performance.benchmark.generator.DiscreteGenerator;
+import voldemort.utils.Props;
+import voldemort.versioning.Versioned;
+
+/**
+ * This class implements a workload whose key size distribution, value size
+ * distribution and key access sequence are loaded from files. Thus it allows
+ * real world workload to be replayed.
+ */
+public class CustomizedWorkload implements IWorkload {
```

Comment:
Customized is too generic of a name, can you please rename it to something else. TraceReplayWorkload or RecordedReplayWorkload or something else ?